Conceptpower+Spring/src/test/java/edu/asu/conceptpower/app/core/impl/RequestsManagerTest.java

```diff
@@ -61,17 +92,51 @@ public void test_getReview_success() {
     
     @Test
     public void test_getReview_failure() {
-        
-        ReviewRequest reviewRequest = new ReviewRequest();
-        reviewRequest.setStatus(ReviewStatus.OPENED); 
-        reviewRequest.setRequester("admin");
-        reviewRequest.setConceptId("WID-10126926-N-05-Einstein");
-        reviewRequest.setRequest("Testing Method2");
-        
-        Mockito.when(dbClient.getReviewRequestForConcept("WID-10126926-N-05-Einstein")).thenReturn(reviewRequest);
+        Mockito.when(dbClient.getAllReviews("WID-10126926-N-05-Thomas")).thenReturn(null);
 
-        ReviewRequest fetchedReview = requestManager.getReview("WID-10126926-N-05-Thomas");
+        ReviewRequest fetchedReview = requestManager.getLatestReview("WID-10126926-N-05-Thomas");
         
         Assert.assertNull(fetchedReview);
     }
+    
+    @Test
+    public void updateReviewTest() {
+        Comment comment = new Comment();     
+        comment.setComment("Sample Comment");
+        
+        ReviewRequest response  = requestManager.updateReview("REVIEW1234-thisissupposedtobesecret2", ReviewStatus.OPENED, comment, OffsetDateTime.now(), "admin");
+        
+        assertNotNull(response);
+        
+        assertEquals("WID-10126926-N-05-Einstein", response.getConceptId());
+        assertEquals("REVIEW1234-thisissupposedtobesecret2", response.getId());
+        assertEquals("Review the Spelling again", response.getRequest());
+        assertEquals("admin", response.getRequester());
+    }
+    
+    @Test
+    public void getAllReviewsTest() {
+        List<ReviewRequest> response = requestManager.getAllReviews("WID-10126926-N-05-Einstein");
+        
+        assertNotNull(response);
+        
+        assertEquals(1, response.size());
+        assertEquals("Review the Spelling again", response.get(0).getRequest());
+        assertEquals("REVIEW1234-thisissupposedtobesecret2", response.get(0).getId());
```

Comment:
since you're using the ids so much all over the place, they should be constants.